### ISSUE TITLE
[Bug report] Return "dropped":true when run curl -X DELETE http://localhost:8090/api/metalakes/metalake_demo/tags/null

### ISSUE THREAD

**Issue by danhuawang** [apache/gravitino#4817]
### Version

main branch

### Describe what's wrong

```
curl -X POST -H "Accept: application/vnd.gravitino.v1+json" \
-H "Content-Type: application/json" -d '{
  "tagsToAdd": ["tag1", "tag2"],
  "tagsToRemove": ["tag3"]
}' http://localhost:8090/api/metalakes/metalake_demo/tags/catalog/catalog_postgres
{"code":0,"names":["tag2"]}
                                                                                                                                         
curl -X GET -H "Accept: application/vnd.gravitino.v1+json" \
http://localhost:8090/api/metalakes/metalake_demo/tags/catalog/catalog_postgres
{"code":0,"names":["tag2"]} 

curl -X DELETE -H "Accept: application/vnd.gravitino.v1+json" \
http://localhost:8090/api/metalakes/metalake_demo/tags/null
{"code":0,"dropped":true}
 
curl -X DELETE -H "Accept: application/vnd.gravitino.v1+json" \
http://localhost:8090/api/metalakes/metalake_demo/tags/null
{"code":0,"dropped":false}
                                                                                                                                            
curl -X GET -H "Accept: application/vnd.gravitino.v1+json" \   
http://localhost:8090/api/metalakes/metalake_demo/tags/catalog/catalog_postgres
{"code":0,"names":[]}                            
```

### Error message and/or stacktrace

N/A

### How to reproduce

N/A

**Comment by jerryshao** [comment]
@danhuawang  What is the issue here?

**Comment by jerryshao** [comment]
Will be fixed by #5235 .
